recipe/meta.yaml

```diff
@@ -93,6 +93,8 @@ requirements:
     - prometheus_client
     # Resolves https://github.com/man-group/ArcticDB/issues/465
     - aws-crt-cpp >=0.19.8
+    # another line to pick up pinning
+    - aws-crt-cpp
```

Comment:
If keeping this comment above is of importance to you, then you can still keep picking up the pinning by having an extra line (without any bounds or pins, so that the bot will pick it up)

Reply:
> If keeping the lower bound for aws-crt-cpp is of importance to you, then you can still keep picking up the pinning by having an extra line (without any bounds or pins, so that the bot will pick it up).

That's a good thing to know. Is this documented somewhere?

With `aws-crt-cpp<0.19.8`, one of the AWS libraries was bugged; if the global pinning moved, we can remove the pin (I'll check that).